### PR TITLE
TMEDIA-363: Add Lead Art block to StoryBook

### PR DESCRIPTION
**Pull request by nschubach**
## Description
Add Lead Art block to StoryBook

## Jira Ticket
- [TMEDIA-363](https://arcpublishing.atlassian.net/browse/TMEDIA-363)

## Acceptance Criteria
Add the following blocks to the Blocks storybook, and set them up for visual regression testing with Chromatic: 
  * Lead Art
    
## Test Steps
1. Checkout this branch `git checkout TMEDIA-363-add-lead-art-to-storybook`
2. Run `npm run storybook` and check the Lead Art category in Blocks.

### Before
Nothing

### After
image

## Dependencies or Side Effects
N/A

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [x] If the amount of work to write unit tests for this change are excessive,
please explain why (so that we can fix it whenever it gets refactored).
- [x] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist
_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

### CODE REVIEW

File: blocks/lead-art-block/index.story.jsx [hunk 1]
```diff
@@ -0,0 +1,506 @@
...
+    }],
+    by: [{
+      byline: 'Scott Suchman for The Washington Post',
+      name: 'Scott Suchman for The Washington Post',
+      type: 'author',
+    }],
+  },
+  subtitle: 'Key Lime Pie story in Voraciously',
+  url: 'https://cloudfront-us-east-1.images.arcpublishing.com/corecomponents/Y5HK2DX6WJHIBAV7VBMSIBFB6M.jpeg',
+  resized_params: {
```
Comment (matthewroach) [inline blocks/lead-art-block/index.story.jsx:22]: Does `LeadArt` require **all** these resized_params?

By only having the ones needed for the block it 
1. Allows us to see what is required for a given block
2. Reduces the overhead of parsing this file

File: blocks/lead-art-block/index.story.jsx [hunk 2]
```diff
@@ -0,0 +1,506 @@
...
+
+export const image = () => {
+  const globalContent = {
+    arcSite: 'story-book',
+    promo_items: {
+      lead_art: leadArtImage,
+    },
+  };
+  return (
+    <LeadArt globalContent={globalContent} />
```
Comment (matthewroach) [inline blocks/lead-art-block/index.story.jsx:142]: The image is not showing within Chromatic - https://www.chromatic.com/test?appId=5f91e4721ccaba0022a10782&id=61032846d790750039b0282d

**Review by matthewroach — CHANGES_REQUESTED** [review]
Couple of small items,

1. Mock data
2. image story is not showing image

File: blocks/lead-art-block/index.story.jsx (hunk 1 above)
Comment (nschubach) [reply blocks/lead-art-block/index.story.jsx:22]: I honestly don't know which ones it needs or how this actually works.  I know that it needs some of them.  I tried paring the options down to what was necessary, but this one got several levels deep into the image stuff and I stopped.  I can look into it.

File: blocks/lead-art-block/index.story.jsx (hunk 2 above)
Comment (nschubach) [reply blocks/lead-art-block/index.story.jsx:142]: Changed out which image this uses.  It was working but now I'm getting an error...

File: blocks/lead-art-block/index.story.jsx (hunk 1 above)
Comment (nschubach) [reply blocks/lead-art-block/index.story.jsx:22]: cut out a lot of these based on what is requested by the Image component

File: blocks/lead-art-block/index.story.jsx [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+  }, {
+    _id: 'PXYWEPTDCJF7JKDLK435PZM7HI',
+    caption: 'Keep your plate colorful by incorporating plenty of fruits and vegetables into your rotation.',
+    credits: {
+      affiliation: [],
+    },
+    subtitle: 'Tastebud Teasers',
+    url: 'https://cloudfront-us-east-1.images.arcpublishing.com/corecomponents/PXYWEPTDCJF7JKDLK435PZM7HI.jpg',
+    resized_params: {
+      '84x56': '6zB-nJo0DflA5m0nuy_Lqp3T38c=filters:format(jpg):quality(70)/',
```
Comment (matthewroach) [inline blocks/lead-art-block/index.story.jsx]: Can we reduce this object aswell?

**Review by matthewroach — DISMISSED** [review]
Are we able to reduce the resized_params on all items?

File: blocks/lead-art-block/index.story.jsx (hunk 3 above)
Comment (nschubach) [reply blocks/lead-art-block/index.story.jsx]: Yep, missed that one.

File: .storybook/alias/content.js [hunk 4]
```diff
@@ -8,7 +8,7 @@ import { simpleListMock } from '../mock-content/simpleList';
 import { smallPromoMock } from '../mock-content/smallPromo';
 
 const featureMocks = {
-	footer: footerContentMock,
+	'footer': footerContentMock,
```
Comment (JackHowa) [inline .storybook/alias/content.js:11]: was this change deliberate?

File: .storybook/alias/content.js [hunk 5]
```diff
@@ -51,79 +51,79 @@ export const useContent = ({ query }) => {
...
+      '274x274': 'MDrtZOPpTHNfaBvyvUpoDRp_Pmw=/fit-in/274x274/filters:quality(70):fill(white):background_color(white)/',
+      '377x377': 'ievAZC23UVlp0pDO9i_vYd4Mzns=/fit-in/377x377/filters:quality(70):fill(white):background_color(white)/',
+      '400x400': '9PtxKndKHARFk6D0ynV-eeWSyow=/fit-in/400x400/filters:quality(70):fill(white):background_color(white)/',
+      '600x600': 'rxxPemZQ3CI0OrPBoSnSHQKPhpA=/fit-in/600x600/filters:quality(70):fill(white):background_color(white)/',
+      '768x768': 'Exeh9lKLfeQ8EY8BFF6MAIWw8f4=/fit-in/768x768/filters:quality(70):fill(white):background_color(white)/',
+      '800x800': 'xzLASjSBuaDaTlfQfKRfNx_TXtQ=/fit-in/800x800/filters:quality(70):fill(white):background_color(white)/',
+      '1024x1024': 'RkTnuW6j5K7QDCOkBuXOQNxF0zM=/fit-in/1024x1024/filters:quality(70):fill(white):background_color(white)/',
+      '1440x1440': 'C01cML7mEmfmz3nF_uKCcmQAsdo=/fit-in/1440x1440/filters:quality(70):fill(white):background_color(white)/',
+      '1600x1600': 'CQcCSFygCQqghFAVoFOYMGePDKg=/fit-in/1600x1600/filters:quality(70):fill(white):background_color(white)/',
```
Comment (JackHowa) [inline .storybook/alias/content.js:126]: looks like this formatting changed as well?

**Review by JackHowa — APPROVED** [review]
looks good with storybook. comments were addressed. some minor formatting changes that look due to autoformatting or a different linting setup

File: .storybook/alias/content.js (hunk 5 above)
Comment (nschubach) [reply .storybook/alias/content.js:126]: yep, just consistency here since these look copy-pasted

File: .storybook/alias/content.js (hunk 4 above)
Comment (nschubach) [reply .storybook/alias/content.js:11]: yeah, I wanted to update it for code consistency.

Comment (JackHowa) [reply .storybook/alias/content.js:11]: ah gotcha

Comment (JackHowa) [reply .storybook/alias/content.js:11]: ah gotcha

File: .storybook/alias/content.js (hunk 5 above)
Comment (JackHowa) [reply .storybook/alias/content.js:126]: ah cool
